identity/manager.go

```diff
@@ -51,6 +54,37 @@ func (idm *identityManager) GetIdentities() []Identity {
 	return ids
 }
 
+func (idm *identityManager) Unlock(identity Identity, passphrase string) error {
+	keystoreManager := idm.keystoreManager
+	accountExisting := identityToAccount(identity)
+
+	account, err := keystoreManager.Find(accountExisting)
```

Comment:
What if I already have "found" identity which I retrieved with `manager.GetIdentity()`?

Reply:
What if you didn't? :D

Reply:
Seems like we have 2 structures:
- `IdentityAddress` or `IdentityId` - just unique string `0x4e878562376372CCa10397245CBc2950d42246e1`, could point to another node
- `Identity` or `IdentityWallet`- which is my controlled identity from my owned keystore. With Lock/Unlock functions